languages/csharp/exercises/concept/arrays/.docs/instructions.md

```diff
@@ -0,0 +1,57 @@
+You're an avid bird watcher that keeps track of how many birds have visited your garden in the last seven days.
+
+You have five tasks, all dealing with the numbers of birds that visited your garden.
+
+### 1. Calculate the total number of visiting birds
+
+Implement the `BirdCount.Total()` method to return the total number of birds that have visited your garden:
+
+```csharp
+int[] dayCounts = { 2, 5, 1, 4, 7, 0 };
```

Comment:
The `dayCounts` array name feels... clunky 🤷‍♂️. Would `dailyCount` or `birdsPerDay` indicate the purpose better?

Reply:
I like `birdsPerDay`!

Reply:
I like it too!